app/components/searchable_collection_component/searchable_collection_component.html.slim

```diff
@@ -1,30 +1,8 @@
-= tag.div(class: classes, **html_attributes) do
+= tag.div(class: classes, **html_attributes.merge({ data: { controller: "searchable-collection" } })) do
```

Comment:
it's better to do `.merge({ data: { controller: "searchable-collection" } })` in the ruby file than in the view